neuralpp/test/quick_tests/symbolic/normalizer_test.py

```diff
@@ -176,22 +185,17 @@ def test_quantifier_normalizer():
                                                                     BasicSummation(int, C,
                                                                                    empty_context & (0 < C) & (C < 10),
                                                                                    if_then_else(B < 5, C, 1)))))
+    BB, CC = sympy.symbols('B C')
+    product = SymPyExpression.from_sympy_object(9 * BB * (BB + CC), {BB: int, CC: int})
```

Comment:
I called `from_sympy_object` here because it's not same as `9 * B * (B+C)`, since SymPy uses a 3-argument multiplication internally. Our overloaded `*` is a 2-argument multiplication.